pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -154,6 +162,7 @@ type ExperimentComponents struct {
 	ENV        []ExperimentENV `json:"env,omitempty"`
 	ConfigMaps []ConfigMap     `json:"configMaps,omitempty"`
 	Secrets    []Secret        `json:"secrets,omitempty"`
+	AppAnnotations []ApplicationAnnotations `json:"appannotations"`
```

Comment:
Building on the same logic described for RunnerAnnotation, we could have: 

ExperimentAnnotations map[string]string `json:"experimentannotations,omitempty"`